core/domain/suggestion_services.py

```diff
@@ -2341,9 +2341,20 @@ def _update_translation_review_stats_models(
     Args:
         translation_review_stats: list(TranslationReviewStats). A list of
             TranslationReviewStats domain objects.
+
+    Raises:
+        Exception. Language is None.
+        Exception. Contributor user ID is None.
+        Exception. Topic ID is None.
     """
     stats_dict = {}
     for stat in translation_review_stats:
+        if stat.language_code is None:
```

Comment:
If we make them non-optional, then why would they be None? This seems self-imposed unless I'm missing something.

Reply:
Removed